schemas/oic.types-schema.json

```diff
@@ -23,6 +23,12 @@
           "type": "string",
           "pattern": "^[A-Za-z]{1,8}(-[A-Za-z0-9]{1,8})*$"
         },
+        "uint8": {
+          "description": "A integer that can be represented as a Byte. an array of uint8 can be represented in CBOR as  Major type 2",
```

Comment:
"can be represented as a Byte" is "is represented as a Byte"?

"can be represented in CBOR as Major type 2" or "is represented in CBOR as Major type 2"

Will this new type need to be treated differently by the CTT?  Will the CTT automatically understand a Property defined as "uint8"?